stdlib/runtime/numbers.gr

```diff
@@ -701,14 +705,14 @@ let numberAddSubFloat32Help = (xval, y, isSub) => {
     let result = if (isSub) WasmF64.sub(xval, yval) else WasmF64.add(xval, yval)
     newFloat64(result)
   } else {
-    let yval = coerceFloat32(y)
+    let yval = coerceNumberToWasmF32(WasmI32.toGrain(y) : Number)
```

Comment:
This coercion back to a Grain number is a bit weird to me.

Reply:
I asked them to do it like this because I thought it was better than having two separate values since `toGrain` and `fromGrain` are just no-ops.